### PR TITLE
[Quest] Fix Suntara Stones Escort

### PR DESCRIPTION
**Pull request by miraco**
Needs the core fixes from https://github.com/cmangos/mangos-tbc/pull/602

https://github.com/cmangos/issues/issues/3240#

### CODE REVIEW

File: Updates/0703_escort.sql [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+(@RELAYID,241000,1,18,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Dorius Stonetender - Despawn Self');
+
+-- Dark Iron Marksman relayscript OnSpawn
+SET @RELAYID := 8338;
+DELETE FROM creature_spawn_data_template WHERE Entry = @RELAYID;
+INSERT INTO `creature_spawn_data_template` (`Entry`, `RelayId`, `Name`) VALUES (@RELAYID, @RELAYID, 'Dark Iron Marksman - RelayScript on Spawn');
+DELETE FROM `dbscripts_on_relay` WHERE id = @RELAYID;
+INSERT INTO dbscripts_on_relay(id, delay, priority, command, datalong, datalong2, datalong3, buddy_entry, search_radius, data_flags, dataint, dataint2, dataint3, dataint4, datafloat, x, y, z, o, speed, condition_id, comments) VALUES
+(@RELAYID,0,0,0,0,0,0,0,0,0,4358,0,0,0,0,0,0,0,0,0,0,'Dark Iron Marksman - Say Text'),
+(@RELAYID,1000,0,15,12198,0,0,8284,30,0,0,0,0,0,0,0,0,0,0,0,0,'Dark Iron Marksman - Cast Spell'),
```
Comment (miraco) [inline Updates/0703_escort.sql:139]: I think thats the only thing thats not working correctly atm.
The spell gets casted from Dark Iron Marksman (animation) and needs to hit Dorius Stonetender.
currently the animation on dark iron marksman is missing but dorius gets hited.
if we use SCRIPT_FLAG_BUDDY_AS_TARGET, dark iron marksman has the cast animation but the spell dont hit dorius anymore so the rp (core OnSpellHit) dont get triggert

**Comment by AnonXS** [comment]
https://github.com/cmangos/tbc-db/commit/03e99200c9909f022daf4db8b65ed40a1e6f1f5b

**Comment by AnonXS** [comment]
Is there maybe also death aura missing at the end, or should the shoot kill him?

grafik

**Comment by miraco** [comment]
https://youtu.be/v6Z72Gk70AI?t=850

No only emote

**Comment by AnonXS** [comment]
i think the spawning dwarf was attackable for me when i tested?

**Comment by miraco** [comment]
I didnt test that, but iirc i couldnt attack the spawned dwarf, can test on wotlk classic ptr later (after raid)

**Comment by miraco** [comment]
https://www.youtube.com/watch?v=xV24oMe4jBY&feature=youtu.be

Can confirm, summoned npc isnt attackable

**Pull request closed without merging**